README.md

```diff
@@ -206,6 +206,7 @@ node {
 ```groovy
 @Test
 void exampleReadFileTest() {
+    helper.addFileExistsMock('fileExists', true)
```

Comment:
shouldn't this be 
`helper.addFileExistsMock('output', true)`?

Reply:
I had meant `fileExists` as in, "this file exists", but I agree that this could be a bit confusing. I'll open a PR to improve the example code.